ee/clickhouse/queries/trends/total_volume.py

```diff
@@ -50,13 +52,22 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team: Team) -> Tup
         else:
 
             if entity.math in [WEEKLY_ACTIVE, MONTHLY_ACTIVE]:
-                content_sql = ACTIVE_USER_SQL.format(
-                    event_query=event_query,
-                    **content_sql_params,
-                    parsed_date_to=trend_event_query.parsed_date_to,
-                    parsed_date_from=trend_event_query.parsed_date_from,
-                    **trend_event_query.active_user_params,
-                )
+                if not team.aggregate_users_by_distinct_id:
+                    content_sql = ACTIVE_USER_SQL.format(
+                        event_query=event_query,
+                        **content_sql_params,
+                        parsed_date_to=trend_event_query.parsed_date_to,
+                        parsed_date_from=trend_event_query.parsed_date_from,
+                        **trend_event_query.active_user_params,
+                    )
+                else:
+                    content_sql = ACTIVE_USER_DISTINCT_ID_SQL.format(
```

Comment:
Nit: It seems like all that's changing in `ACTIVE_USER_SQL` is `person_id` -> `distinct_id`. 

Since the string isn't used elsewhere, can just make the `aggregate_by` a parameter, and do something like:

```
aggregator = 'distinct_id' if team.aggregate_users_by_distinct_id else 'person_id'
```